eng/SnippetGenerator/CSharpProcessor.cs

```diff
@@ -12,38 +11,59 @@ namespace SnippetGenerator
     public class CSharpProcessor
     {
         private static readonly string _snippetFormat = "{3} <code snippet=\"{0}\">{1}{2} </code>";
-        private static readonly Regex _snippetRegex = new Regex("^(?<indent>\\s*)\\/{3}\\s*<code snippet=\\\"(?<name>[\\w:]+)\\\">.*?\\/{3}\\s*<\\/code>",
+        private static readonly string _snippetExampleFormat = "{3} <example>{1}{3} <code snippet=\"{0}\" example=\"true\">{1}{2} </code>{1}{3} </example>";
```

Comment:
This format doesn't seem right.

Reply:
Yeah, I left the `example=true` bit in. Fixed.